src/sdb.c

```diff
@@ -717,9 +723,9 @@ SDB_API int sdb_expire_set(Sdb* s, const char *key, ut64 expire, ut32 cas) {
 
 SDB_API ut64 sdb_expire_get(Sdb* s, const char *key, ut32 *cas) {
 	SdbKv *kv;
-	ut32 hash = sdb_hash (key);
-	kv = (SdbKv*)ht_lookup (s->ht, hash);
-	if (kv && *kv->value) {
+	bool found;
```

Comment:
initialize to false, if ht_find_kvp have a bug (didnt chkd src) it may fail.. but... why do we need a &found when the function is returning the result which must be != NULL in case of being valid?